pkg/cli/webhook/gitlab.go

```diff
@@ -44,7 +44,7 @@ func (gl *gitLabConfig) Run(_ context.Context, opts *Options) (*response, error)
 }
 
 func (gl *gitLabConfig) askGLWebhookConfig(controllerURL, apiURL string) error {
-	msg := "Please enter the project ID for the repository you want to be configured, \n  project ID refers to an unique ID shown at the top of your GitLab project :"
+	msg := "Please enter the project ID for the repository you want to be configured, \n  project ID refers to an unique ID (e.g. 123456789) shown at the top of your GitLab project :"
```

Comment:
should we add an actual id instead of just 1...9?
we can use `34405323` it is pac e2e test repo.. which is public

Reply:
Updated